ngi_reports/ngi_reports.py

```diff
@@ -10,39 +10,92 @@
 import json
 import os
 import markdown
+import sys
 
 from ngi_reports import __version__
 from ngi_reports.log import loggers
 from ngi_reports.utils import config as report_config
 from ngi_reports.utils.entities import Project
 
-LOG = loggers.minimal_logger('NGI Reports')
+LOG = loggers.minimal_logger("NGI Reports")
 
 ## CONSTANTS
 # create choices for report type based on available report template
-allowed_report_types = [ fl.replace(".md","") for fl in os.listdir(os.path.realpath(os.path.join(os.path.dirname(__file__), os.pardir, 'data', 'report_templates'))) ] + ['ign_aggregate_report']
+allowed_report_types = [
+    fl.replace(".md", "")
+    for fl in os.listdir(
+        os.path.realpath(
+            os.path.join(
+                os.path.dirname(__file__), os.pardir, "data", "report_templates"
+            )
+        )
+    )
+] + ["ign_aggregate_report"]
 
-def make_reports (report_type, working_dir=os.getcwd(), config_file=None, **kwargs):
+
+def proceed_or_not(question):
+    yes = set(["yes", "y", "ye", "Y", "YES", "YE", "Yes"])
+    no = set(["no", "n", "N", "NO", "No"])
+    sys.stdout.write("{}".format(question))
```

Comment:
I wonder if stderr is better than stdout? I guess it doesn't matter though since the stdout is most likely not piped to something else...

Reply:
`proceed_or_not` has been 1 to 1 copied from other code in our code base. I can of course change it, but then we should probably change it there, too. You want me to do it?